coordinator/src/services/environment.ts

```diff
@@ -0,0 +1,4 @@
+const environment = process.env.ENVIRONMENT
+
+export const isProd = (): boolean => environment === "prod"
```

Comment:
Is there a reason these need to be functions instead of variables

Reply:
easy way to fake prod mode in tests - mock out return value of these functions. easier than changing env variables per test